### ISSUE TITLE
Ineffectual branch condition

### ISSUE THREAD

**Issue by justincely** [ZachSellman/pokereddit_bot#6]
Starting [here](https://github.com/ZachSellman/pokereddit_bot/blob/main/pokemon_scanner.py#L55) you have a condition that doesn't actually do anything.

```python
                if pokemon in comment.body.lower():
                    add_pokemon(pokemon)
                else:
                    continue
```

is functionally equivalent to 

```python
                if pokemon in comment.body.lower():
                    add_pokemon(pokemon)
```

because you're already at the end of the loop, so you're going to the next iteration regardless of whether you call `continue`.  If this _wasn't_ at the end of the loop, and you were using `continue` to skip some chunk of code that you no longer wanted to process through, then this would do something.  

Basically, omit clauses that are just the default behavior of the loop/code/function.

**Comment by ZachSellman** [comment]
Yeah I've caught myself doing this a few times. The first class I took on python they had me write every initial for loop like this and then afterwards went "You actually DON'T need to do this all the time!". It's been removed, should see it on the next contribution.
